hera_mc/geo_location.py

```diff
@@ -208,7 +212,7 @@ def is_in_connections(args, station_name, check_if_active=False):
     ------------
     args:  needed arguments to open database and set date/time
     station_name:  string name of station
-    check_if_active:  boolean flag to check if active
+    check_if_active:  either False to not check/care if active, or astropy Time to check
```

Comment:
I've done a major overhaul in another branch, which handles some of this (and moves modules in other files...)